doc/quick-start.md

```diff
@@ -76,7 +82,11 @@ Memory profiling requires extra dependencies: `pip install matplotlib psutil`
 ```
 
 ```{code-cell} ipython3
-_ = execute_notebook("nb.ipynb", "output.ipynb", profile_memory=True)
+:tags: []
```

Comment:
are these empty tags needed?

Reply:
removed them. I think jupyter kept them since I added one cell and then deleted it